### PR TITLE
Code review suggestions.

### PR DESCRIPTION
**Pull request by Odogwufu**
Description of the project:
App for helping people meet up with each other and plan coordinated group meetings. Users can see the calendar and users can see the contact list.

Was the program available in UC GitHub on time?
Yes, effiencient code was done in the activity_main.xml file

Is the program documented well enough?
Yes, it is documented well enough and follows the coding conventions

does the program build
yes the build was successful

rationale behind changes
One change that I made was creating a service package and adding a service class mainly because it helps keep the code more organized and it's easier to find what you're looking for. Another change I made is a created a dto package and in that package, I made 2 classes date and a calendar. This class will connect to the activity.xml file when fully implemented. Lastly, I added some private variables in the main activity class file because they would be needed when the project is fully coded.

Links to three commits I made:
https://github.com/sharmrb/TimeMan/commit/87c3dc09ff5aecc49e766cfd9f924add8f1c71cd
https://github.com/sharmrb/TimeMan/commit/a3962b12f50055db87ffe6d576375381a31bedec
https://github.com/sharmrb/TimeMan/commit/aae61560cdd04f32122ca9740e5b33963ba83534

Technical concepts I learned
1. how to create a calendar using image view
2. How constraint layout works In kotlin
3. I also learned how to use vector images/icons

### CODE REVIEW

File: app/src/main/java/com/example/plan2meet/MainActivity.kt [hunk 1]
```diff
@@ -4,6 +4,8 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var day:String
```
Comment (Odogwufu) [inline app/src/main/java/com/example/plan2meet/MainActivity.kt:7]: added variables per the suggestions here:https://kotlinlang.org/docs/basic-syntax.html

File: app/src/main/java/com/example/plan2meet/Service/CalendarService.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.plan2meet.Service
+
```
Comment (Odogwufu) [inline app/src/main/java/com/example/plan2meet/Service/CalendarService.kt:2]: created calendar service class here:https://kotlinlang.org/docs/basic-syntax.html

File: app/src/main/java/com/example/plan2meet/MainActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/example/plan2meet/MainActivity.kt:7]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.
--
Is String the best way to represent a day and month?

File: app/src/main/java/com/example/plan2meet/Service/CalendarService.kt (hunk 2 above)
Comment (discospiff) [reply app/src/main/java/com/example/plan2meet/Service/CalendarService.kt:2]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: app/src/main/java/com/example/plan2meet/dto/Calendar.kt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.plan2meet.dto
+
+class Calendar {
+}
```
Comment (discospiff) [inline app/src/main/java/com/example/plan2meet/dto/Calendar.kt:4]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: app/src/main/java/com/example/plan2meet/dto/date.kt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.example.plan2meet.dto
+
+class date {
+}
```
Comment (discospiff) [inline app/src/main/java/com/example/plan2meet/dto/date.kt:4]: This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: app/src/main/java/com/example/plan2meet/dto/date.kt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package com.example.plan2meet.dto
+
+class date {
```
Comment (discospiff) [inline app/src/main/java/com/example/plan2meet/dto/date.kt:3]: Incorrect syntax for a class.  Also, ambiguous.

**Comment by discospiff** [comment]
additions are likely needed for the project at some point, but are new, and sometimes incomplete, functionality.  New functionality is better in a separate branch until fully developed, as features should not be merged to main/master until the meet the team's Definition of Done.  Code review branches should minimize technical debt, and thus, be merge-able to master immediately.  So, if the team desires these features, I'd keep the branch open, and iterate over the feature until it is complete.

**Comment by WillTaylor2352** [comment]
We have discussed the proposed changes and have decided to not proceed further with this code review.

**Pull request closed without merging**